src/cli/init_command.zig

```diff
@@ -822,14 +822,12 @@ pub const InitCommand = struct {
                 }
 
                 if (fields.entry_point.len > 0 and !did_load_package_json) {
-                    Output.pretty("\nTo get started, run:\n\n\t", .{});
-                    if (strings.containsAny(
-                        " \"'",
-                        fields.entry_point,
-                    )) {
-                        Output.prettyln("<cyan>bun run {any}<r>", .{bun.fmt.formatJSONStringLatin1(fields.entry_point)});
+                    Output.pretty("\nTo get started, run:\n\n    ", .{});
```

Comment:
```suggestion
                    Output.pretty("\nTo get started, run:\n\n      ", .{});
```
The commands at the end of this file use 6 spaces

Reply:
then i should shorten those as well